src/widgets/dropdown/template.tpl

```diff
@@ -1,14 +1,44 @@
 <ul class="CDB-Dropdown-list CDB-Text CDB-Size-medium">
   <li class="CDB-Dropdown-item">
-    <button class="CDB-Dropdown-link js-toggle">Toggle widget</button>
+    <button class="CDB-Dropdown-link js-toggleCollapsed">
+      Toggle widget
+        <div>
+        <% if (collapsed) { %>
```

Comment:
We were getting linting errors when we wrote the condition of the template inside of an HTML tag. Writing something like <div <% if (this) { that } %> class="wait-what"></div> aborted the compilation of the code.